docs(nav): Implement feedback for Nav, Introduce new top level items and restructure

parth-srivastav opened the pull request:
Implements some of the feedback recieved on the first iteration of the Nav changes.
Changes Include
- Moving Spinnaker Plugin into Integrations
- Making Webhooks and Canary Analysis Top level items
- Restructuring Canary and Blue/Green into a new category called deployment strategies
- Renaming Access Management to Identity and Access Management

No Content Changes have been made thus far. 

Deploy preview https://deploy-preview-18--cdaas-docs.netlify.app/

parth-srivastav committed:
Moved Spinnaker into Integrations and moved getting started section out on its own [WIP]

parth-srivastav committed:
Revert "Moved Spinnaker into Integrations and moved getting started section out on its own [WIP]"

This reverts commit 59991b700556ab7383f8592808068f17ae759032.

parth-srivastav committed:
Move Spinnaker plugin into Integrations

parth-srivastav committed:
Fix Spinnaker plugin moved to Integration, links Fixed

parth-srivastav committed:
Move Webhooks Out Of Integrations

parth-srivastav committed:
Change Nav: Newtop level items, moved strategies

aimeeu commented:
Left menu on Parth's commit:
![image](https://github.com/armory/docs-cdaas/assets/26799583/fc9d344a-23c4-4a0b-bda0-b5cbebe0f8f1)

Talked to Parth and came up with this:
Remote Network Agent
Deployment
Canary Analysis
Webhooks
Traffic Management
Integrations
Identity & Access Management

aimeeu committed:
change left nav to put RNA at top of features

Switch to foldable menu b/c other menu weird behavior
Add Overview page to most root-level sections
Change section landing page to list pages

y

aimeeu committed:
added overview page to deployment strategies

cleaned up categories, tags for new structure

aimeeu committed:
fixed broken image links in quickstart
